diff --git a/src/tl_templates/cuda/copy_sm100.h b/src/tl_templates/cuda/copy_sm100.h
--- a/src/tl_templates/cuda/copy_sm100.h
+++ b/src/tl_templates/cuda/copy_sm100.h
@@ -167,7 +167,7 @@
 }
 
 __device__ __forceinline__ unsigned long long
-pack_float16x4(const half x, const half y, const half z, const half w) {
+pack_float16x4(const half_t x, const half_t y, const half_t z, const half_t w) {
   unsigned long long v0 = *((unsigned short *)&x);
   unsigned long long v1 = *((unsigned short *)&y);
   unsigned long long v2 = *((unsigned short *)&z);
